src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -144,10 +145,8 @@ const WorkspaceDashboard = _.flow(
 }, ref) => {
   // State
   const [submissionsCount, setSubmissionsCount] = useState(undefined)
-  const [storageCostEstimate, setStorageCostEstimate] = useState(undefined)
-  const [storageCostEstimateUpdated, setStorageCostEstimateUpdated] = useState(undefined)
+  const [storageCost, setStorageCost] = useState(undefined)
```

Comment:
I am consolidating storageCost and storageCostEstimateUpdated because the information is always fetched together. Same for bucketSize.

FYI @MatthewBemis